bin/parse

```diff
@@ -13,7 +14,8 @@ if (!fileName) {
 
 switch (args.paradigm) {
   case 'oo':
-    // load and use object oriented parser
+    const insaneParser = new ooParser( fileContents )
```

Comment:
Please check on your naming conventions, is this explicit enough of a name for this variable?